### ISSUE TITLE
single_to_multi_fast5 compression is not working ?

### ISSUE THREAD

**Issue by svennd** [nanoporetech/ont_fast5_api#38]
In the latest version I can't seem to compress & go from single to multiple;
```
single_to_multi_fast5  --version
3.1.3
```

using : 
```
 single_to_multi_fast5 -i input -s output.mod -t 16 --recursive -c vbz
```

It does not give an error, but going from single -> multiple -> compressed multiple gives compression;

sizes :
```
187M    ori (original)
134M    ori.uncompressed (single -> multi)
134M    ori.modC (single -> multi + vbz)
93M     ori.compress (multi -> vbz)
```

Am I doing something wrong ? Obv, i can do it in two steps, but since you where so nice to make it in 1 command possible, it would be nice.

**Comment by fbrennen** [comment]
Hi @svennd -- should be all fixed now (in version 3.1.4). Have a try!

**Comment by svennd** [comment]
I tried, it works now, thanks for the help !

Op vr 12 jun. 2020 om 18:17 schreef Forrest Brennen <
notifications@github.com>:
